src/main/java/core/basesyntax/service/FruitDtoService.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.service;
+
+import core.basesyntax.products.FruitDto;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FruitDtoService {
```

Comment:
```suggestion
public class FruitDtoMapper {
```
this is more a Mapper class than a Service class

Reply:
I have googled about mapper didn't know before about this, you are right, changed to FruitDtoMapper.